crates/api/src/post.rs

```diff
@@ -285,3 +285,20 @@ impl Perform for SavePost {
     Ok(PostResponse { post_view })
   }
 }
+
+#[async_trait::async_trait(?Send)]
+impl Perform for GetSiteMetadata {
+  type Response = GetSiteMetadataResponse;
+
+  async fn perform(
+    &self,
+    context: &Data<LemmyContext>,
+    _websocket_id: Option<ConnectionId>,
+  ) -> Result<GetSiteMetadataResponse, LemmyError> {
+    let data: &Self = self;
```

Comment:
Why do you rename `self` in every API method? For me its quite confusing, because it doesnt actually do anything.

Reply:
Just history, these used to be very different impls. If someone wanted to go through and change all these to self, it wouldn't hurt anything, but data to me is more clear as its describing that its the `Form` data.